mcp/pacemaker.sysconfig.m4

```diff
@@ -1,8 +1,10 @@
+ifelse(SUPPORT_CMAN, 1,
 # Turn on special handling for CMAN clusters in the init script.
 # Without this, fenced (and by inference, cman) cannot reliably be made
 # to shut down.
 # PCMK_STACK=cman
 
+)dnl
```

Comment:
First, `m4` preprocessing and `configure` distilling `AC_CONFIG_FILES` are orthogonal and can be combined smoothly.  The latter indeed works great for substituting line-less values, but longer multi-line excepts are better handled by other means (just as you don't want code parts substituted likes that, but rather want to have an overall, multi-context view within a single file and clear idea when this or that part is effectively used).

Adding to you incompleteness note, there's much more possibly missing to some users:
- upstart job for pacemaker_remote
- init script for crm_mon
- ...